addons/medical/ACE_Medical_Treatments.hpp

```diff
@@ -444,9 +411,7 @@ class ACE_Medical_Actions {
             items[] = {"ACE_samSplint"};
             treatmentTime = 10;
             callbackSuccess = QUOTE(DFUNC(treatmentSAMSplint));
-            treatmentType = "Others";
-            treatmentDisplay = "Applying SAM Splint";
-            condition = "[_this select 1, _this select 2] call ace_medical_fnc_canTreatSAMSplint";
+            condition = QUOTE([ARR_2(_this select 1, _this select 2)] call FUNC(canTreatSAMSplint));
```

Comment:
That's still the same as before ^^ Just harder to read now.

Reply:
As I commented before, this is easier to read and does not suffer from the performance waste
`condition = QFUNC(canTreatSAMSplint);`

Reply:
Well it's now fixed that way. BTW, isn't QFUNC different from QUOTE(DFUNC(..))? AFAIK DFUNC has something different from FUNC about debugging.

Reply:
And would it be better to accept variables via `_this select 1` rather than `params`? the 0th item isn't used, so accepting that via `params` doesn't look so optimized.

Reply:
>BTW, isn't QFUNC different from QUOTE(DFUNC(..))?

https://github.com/acemod/ACE3/blob/master/addons/main/script_macros.hpp#L15

>And would it be better to accept variables via _this select 1 rather than params?
no it won't. 2 binary commands are worse than 1 unary command. If you name the first variable `""` it's skipped very fast.